src/systems/systemItemsDialog.component.tsx

```diff
@@ -276,6 +278,21 @@ const SystemItemsDialog = React.memo((props: SystemItemsDialogProps) => {
               />
             </Grid>
             <Grid size={12}>
+              {parentSystemId &&
+                selectedItems.length !== 0 &&
+                !isSelectedRulesLoading &&
+                !systemsDataLoading && (
+                  <MRTTopTableAlert
+                    title="Item Moving Rule Applied"
+                    showInfoTooltip
+                    infoTooltipTitle={
+                      selectedRules && selectedRules[0]
+                        ? `The items usage statuses will be updated to ${selectedRules[0].dst_usage_status?.value}, as defined by the rules`
+                        : 'The items usage statuses we remain the same, as defined by the rules'
```

Comment:
```suggestion
                        ? `The items usage statuses will be updated to ${selectedRules[0].dst_usage_status?.value}, as defined by the rules`
                        : 'The items usage statuses will remain the same, as defined by the rules'
```
The first of these can probably be the same as the items dialog one or vice versa.  `The item’s usage status will be updated to ${selectedRules?.[0]?.dst_usage_status?.value ?? selectedItem?.usage_status}, as defined by the rules`
The apostrophe used is a wierd one though, might have come from word or something, I get
<img width="1012" height="69" alt="image" src="https://github.com/user-attachments/assets/7fc88744-f145-4f6a-8e70-81a98bd1d636" />

Reply:
This does have to be slightly different as you can move many in systems item dialog and you can only move one in the items dialog